app/views/admin/roles_service/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for :page_title, construct_page_title(t('hyrax.admin.roles_service_jobs.header'), t('hyku.admin.title')) %>
+<% provide :page_header do %>
+  <h1><span class="fa fa-wrench" aria-hidden="true"></span>
+  <%= t('hyrax.admin.roles_service_jobs.header') %></h1>
+<% end %>
+
+<div class='panel panel-default'>
+  <div class='panel-body'>
+    <div class='table-responsive'>
+      <table class='table table-striped datatable'>
+          <!-- <thead>
```

Comment:
The labels columns show 3 headers; but only two columns in the body.  Is that intentional?

Reply:
oops no, not intentional. I'll fix it.